packages/react-textarea/src/components/TextArea/TextArea.types.ts

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type TextAreaSlots = {
+  /**
+   * Wrapper element used for displaying the borders for TextArea. This wrapper is needed due to the focus
+   * indicator border animation. For more information, see Spec.md
+   *
+   * The root only received a `className` and `style`. All other props are applied to the `textArea` slot.
+   */
+  root: NonNullable<Slot<'span'>>;
+
+  /**
+   * The `<textarea>` element. This is the primary slot, all native props and ref will be applied to this slot.
+   */
+  textArea: NonNullable<Slot<'textarea'>>;
+};
+
+type TextAreaCommons = {
+  /**
+   * Which direction the TextArea is allowed to be resized.
+   *
+   * @defaultvalue none
```

Comment:
We are very inconsistent about this but I think the correct thing to use is `@default` instead of `@defaultvalue`

```suggestion
   * @default none
```